TOS.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>School Chat - Rules & TOS</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
```

Comment:
Should we just create a new style.css file just for this? or just try to use as much as we can from the main style.css file?